frontend/src/components/activity/ScheduleEntry.vue

```diff
@@ -283,6 +298,13 @@ export default {
     scheduleEntries() {
       return this.activity.scheduleEntries().items
     },
+    activityName() {
+      return (
```

Comment:
maybe we already have this logic somewhere.
can be refactored in another PR.

Reply:
Is used differently each time. (sometimes with CategoryChip, sometimes without)
So I'm not sure if this even makes sense. Maybe as a component?